### PR TITLE
Update jvm-toxcore-c to fix building with openjdk-11-jdk-headless

### PR DESCRIPTION
**Pull request by robinlinden**
Prior to this commit, openjdk-11-jdk is required as only it includes the
Java AWT Native Interface that's searched for.

### CODE REVIEW

**Review by roihershberg — APPROVED** [review]
I don't understand any of those changes that are related to Tox4j so I'm approving it.

**Comment by robinlinden** [comment]
Fair enough! I have a few more coming up, I think, but can merge those without review if you prefer. :P If I break anything, CI will block merging anyway. :x

**Comment by roihershberg** [comment]
Yeah you can merge them without my approval if they are related to Tox4j as currently I don't understand those things. Thank you for asking me to review it though!

**Comment by robinlinden** [comment]
No worries! There are some tasks that need doing in tox4j if you feel like messing around with it at some point. :P Just let me know! :D

**Comment by roihershberg** [comment]
OK. Thanks!

**Pull request merged**
